src/core/features/user/user.module.ts

```diff
@@ -51,6 +52,21 @@ export const CORE_USER_SERVICES: Type<unknown>[] = [
 
 export const PARTICIPANTS_PAGE_NAME = 'participants';
 
+const appRoutes: Routes = [
+    {
+        path: 'user',
+        children: [
+            {
+                path: 'completeprofile',
+                loadChildren: () => import('./pages/complete-profile/complete-profile.module')
+                    .then(m => m.CoreUserCompleteProfilePageModule),
+            },
+        ],
+        canActivate: [CoreRedirectGuard],
+        canLoad: [CoreRedirectGuard],
+    },
+];
```

Comment:
```suggestion
const appRoutes: Routes = [
    {
        path: 'user',
        loadChildren: () => import('@features/user/user-app-lazy.module').then(m => m.CoreUserAppLazyModule),
    },
];
```

Instead of creating a page module, I think it'd be better to create a lazy module instead. In practice, it's very similar, but conceptually I think it's better because we only want to define page modules for pages that are used in multiple routes (which is not the case here).